docs/how-to/use-trained-models.txt

```diff
@@ -0,0 +1,197 @@
+instance of :class:`Checkpoint <determined.experimental.Checkpoint>`
+representing the checkpoint from an experiment with the best validation metric.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  checkpoint = Determined().get_experiment(id).top_checkpoint()
+
+Checkpoints can be sorted by any metric, by using ``sort_by``, which states
+which metric to use, and ``smaller_is_better``, which defines whether to sort
+the checkpoints in ascending or descending order with respect to the
+``sort_by`` metric.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  checkpoint = Determined().get_experiment(id).top_checkpoint(sort_by="accuracy", smaller_is_better=False)
+
+You may also query the top n checkpoints to receive a sorted list of
+checkpoints using the ``top_n_checkpoints`` method. This method also accepts
+``sort_by`` and ``smaller_is_better`` arguments.
+
+The following snippet returns the top 5 checkpoints for a specified experiment.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  checkpoints = Determined.get_experiment(id).top_n_checkpoints(5)
+
+
+The :class:`TrialReference <determined.experimental.TrialReference>` is used for
+more fine grained control over checkpoint selection. It contains a
+``top_checkpoint`` method which mirrors the above ``top_checkpoint`` for an
+experiment. It also contains ``select_checkpoint`` which offers 3 ways to query
+checkpoints.
+
+#. ``best``: returns the best checkpoint based on validation metrics as discussed above. When using ``best``, ``smaller_is_better`` and ``sort_by`` are also accepted.
+#. ``latest``: returns the most recent checkpoint for the trial.
+#. ``uuid``: returns the checkpoint for the trial that matches the specified uuid.
+
+
+The following snippet showcases how to use the different modes for selecting
+checkpoints.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  trial = Determined.get_trial(id)
+
+  best_checkpoint = trial.top_checkpoint()
+
+  most_accurate_checkpoint = trial.select_checkpoint(
+    best=True,
+    sort_by="accuracy",
+    smaller_is_better=False
+  )
+
+  most_recent_checkpoint = trial.select_checkpoint(latest=True)
+
+  specific_checkpoint = trial.select_checkpoint(uuid="uuid-for-checkpoint")
+
+Using the Checkpoint class
+--------------------------
+The :class:`Checkpoint <determined.experimental.Checkpoint>` class can both
+download the checkpoint from persistent storage and load it into memory in a
+python process.
+
+The following snippet downloads a checkpoint from persistent storage to the
+local file system.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  checkpoint = Determined().get_experiment(id).top_checkpoint()
+  checkpoint_path = checkpoint.download()
+
+The ``load`` method of the :class:`Checkpoint
+<determined.experimental.Checkpoint>` class downloads the checkpoint if it
```

Comment:
Suggestion, could use a couple of oxford commas.
`downloads the checkpoint, if it does not exist, and then...`